### DISCUSSION TITLE
Error while trying to do transaction

### DISCUSSION THREAD

**Discussion by shivakumarboga** [0xPolygon/polygon-edge#674] [Q&A]
Hi team,

We are getting below error while trying to do a transfer of 1 xDAI from one account to another.

Error details:
[ERROR] polygon.dispatcher: failed to dispatch: method=eth_estimateGas err="unable to apply transaction even for the highest gas limit 5242880: invalid signature"

We have set the price limit as below while starting the node.
![image](https://user-images.githubusercontent.com/110447814/183050150-9c49fbb9-b0da-4da3-b283-de3db53e7e18.png)

<img width="227" alt="image" src="https://user-images.githubusercontent.com/110447814/183050490-24f5a9a0-0882-4736-aedf-8bfb2a3bf2b6.png">

**Comment by laviniat1996** [comment]
Hello @shivakumarboga ,
Please check if the transaction sent is signed on the clientside.
